# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpChannelChild.h
# Commit: e6c12eec1cef
# Full Hash: e6c12eec1cef32987b7a0b79ccf4420ee097b868
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P4 Remove On[Start|Stop]Request/OnTransportAndData in PHttpChannel, r=mayhemer,nika,necko-reviewers
#   
#   We move OnStartRequest from PHttpChannel to PHttpBackgroundChannel, thus adjusting
#   message-metadata.ini
#   
# ==============================================================================

diff -r d59b42f58f86 -r e6c12eec1cef netwerk/protocol/http/HttpChannelChild.h
--- a/netwerk/protocol/http/HttpChannelChild.h	Tue Jun 30 00:32:53 2020 +0000
+++ b/netwerk/protocol/http/HttpChannelChild.h	Tue Jun 30 00:33:01 2020 +0000
@@ -132,20 +132,6 @@
   nsresult CrossProcessRedirectFinished(nsresult aStatus);
 
  protected:
-  mozilla::ipc::IPCResult RecvOnStartRequest(
-      const nsHttpResponseHead& aResponseHead, const bool& aUseResponseHead,
-      const nsHttpHeaderArray& aRequestHeaders,
-      const HttpChannelOnStartRequestArgs& aArgs) override;
-  mozilla::ipc::IPCResult RecvOnTransportAndData(
-      const nsresult& aChannelStatus, const nsresult& aTransportStatus,
-      const uint64_t& aOffset, const uint32_t& aCount,
-      const nsCString& aData) override;
-
-  mozilla::ipc::IPCResult RecvOnStopRequest(
-      const nsresult& aChannelStatus, const ResourceTimingStructArgs& aTiming,
-      const TimeStamp& aLastActiveTabOptHit,
-      const nsHttpHeaderArray& aResponseTrailers,
-      nsTArray<ConsoleReportCollected>&& aConsoleReports) override;
   mozilla::ipc::IPCResult RecvFailedAsyncOpen(const nsresult& status) override;
   mozilla::ipc::IPCResult RecvRedirect1Begin(
       const uint32_t& registrarId, const URIParams& newURI,